packages/design-system/src/__generated__/figma-design-tokens.ts

```diff
@@ -493,28 +535,31 @@ export const color = {
   foregroundOverwrittenFlexUi: "#dc3d43",
   foregroundInversePrimary: "#ff22ae",
   foregroundLinkNotVisited: "#006adc",
```

Comment:
@taylornowotny this already exists as foregroundLink

Reply:
It was renamed

Reply:
but foregroundLink exists too

Reply:
also it would make sense to have foregroundLink as a default, meaning not visited and then a visited one

Reply:
I renamed foreground.link to foreground.link-hover.main  so it should not exist any more. If it does it is a duplicated.

I did not make a 'visited' color but you can ask Mark to make it if you want.